src/vecgf2.c

```diff
@@ -178,235 +178,22 @@ Obj AddCoeffsGF2VecGF2Vec (
-    case 15:     dothework(sptr, dptr, 15, dend); break;
-    case 16:     dothework(sptr, dptr, 16, dend); break;
-    case 17:     dothework(sptr, dptr, 17, dend); break;
-    case 18:     dothework(sptr, dptr, 18, dend); break;
-    case 19:     dothework(sptr, dptr, 19, dend); break;
-    case 20:     dothework(sptr, dptr, 20, dend); break;
-    case 21:     dothework(sptr, dptr, 21, dend); break;
-    case 22:     dothework(sptr, dptr, 22, dend); break;
-    case 23:     dothework(sptr, dptr, 23, dend); break;
-    case 24:     dothework(sptr, dptr, 24, dend); break;
-    case 25:     dothework(sptr, dptr, 25, dend); break;
-    case 26:     dothework(sptr, dptr, 26, dend); break;
-    case 27:     dothework(sptr, dptr, 27, dend); break;
-    case 28:     dothework(sptr, dptr, 28, dend); break;
-    case 29:     dothework(sptr, dptr, 29, dend); break;
-    case 30:     dothework(sptr, dptr, 30, dend); break;
-    case 31:     dothework(sptr, dptr, 31, dend); break;
-#ifdef SYS_IS_64_BIT
-    case 32:     dothework(sptr, dptr, 32, dend); break;
-    case 33:     dothework(sptr, dptr, 33, dend); break;
-    case 34:     dothework(sptr, dptr, 34, dend); break;
-    case 35:     dothework(sptr, dptr, 35, dend); break;
-    case 36:     dothework(sptr, dptr, 36, dend); break;
-    case 37:     dothework(sptr, dptr, 37, dend); break;
-    case 38:     dothework(sptr, dptr, 38, dend); break;
-    case 39:     dothework(sptr, dptr, 39, dend); break;
-    case 40:     dothework(sptr, dptr, 40, dend); break;
-    case 41:     dothework(sptr, dptr, 41, dend); break;
-    case 42:     dothework(sptr, dptr, 42, dend); break;
-    case 43:     dothework(sptr, dptr, 43, dend); break;
-    case 44:     dothework(sptr, dptr, 44, dend); break;
-    case 45:     dothework(sptr, dptr, 45, dend); break;
-    case 46:     dothework(sptr, dptr, 46, dend); break;
-    case 47:     dothework(sptr, dptr, 47, dend); break;
-    case 48:     dothework(sptr, dptr, 48, dend); break;
-    case 49:     dothework(sptr, dptr, 49, dend); break;
-    case 50:     dothework(sptr, dptr, 50, dend); break;
-    case 51:     dothework(sptr, dptr, 51, dend); break;
-    case 52:     dothework(sptr, dptr, 52, dend); break;
-    case 53:     dothework(sptr, dptr, 53, dend); break;
-    case 54:     dothework(sptr, dptr, 54, dend); break;
-    case 55:     dothework(sptr, dptr, 55, dend); break;
-    case 56:     dothework(sptr, dptr, 56, dend); break;
-    case 57:     dothework(sptr, dptr, 57, dend); break;
-    case 58:     dothework(sptr, dptr, 58, dend); break;
-    case 59:     dothework(sptr, dptr, 59, dend); break;
-    case 60:     dothework(sptr, dptr, 60, dend); break;
-    case 61:     dothework(sptr, dptr, 61, dend); break;
-    case 62:     dothework(sptr, dptr, 62, dend); break;
-    case 63:     dothework(sptr, dptr, 63, dend); break;
-#endif
-    default:  Pr("Illegal shift %i", cbits, 0);
-      SyExit(2);
-    }
+  soff = (smin-1) %BIPEB;
+  doff = (dmin-1) %BIPEB;
```

Comment:
Either add spaces after `%` or remove the ones before... Or just use `clang-format`?

(If you add spaces, then perhaps also add them around the `/` in the next two lines?)